### PR TITLE
Us 2

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

**Comment by RogowskaMalgorzata** [comment]
Ok

**Comment by wojciechh-kainos** [comment]
This is the navigation functionality? Should it not be merged from a new branch?

File: src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html [hunk 1]
```diff
@@ -16,34 +21,16 @@
...
-
-        table, td, tr {
-            border: 1px solid black;
-        }
-        .available-slot {
-            background-color: #87D37C;
-        }
-
-    </style>
+    <div>
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html:32]: Both the button and alert should probably be moved to the top of the page

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 2]
```diff
@@ -1,35 +1,52 @@
 define(['angular', 'application/interviewer/tdprInterviewerModule'], function(angular, tdprInterviewerModule) {
-  tdprInterviewerModule.controller("tdprInterviewerHomeController", function($scope, tdprSlotsService, $filter, $stateParams) {
-    $scope.slotTimes = ["9.00-9.30", "9.30-10.00", "10.00-10.30", "10.30-11.00", "11.00-11.30", "11.30-12.00", "12.00-12.30",
-     "12.30-13.00", "13.00-13.30", "13.30-14.00", "14.00-14.30", "14.30-15.00", "15.00-15.30", "15.30-16.00", "16.00-16.30","16.30-17.00", "17.00-17.30", "17.30-18.00"];
+  tdprInterviewerModule.controller("tdprInterviewerHomeController", function($scope, tdprSlotsService, $filter, $stateParams,$state,$timeout) {
+    $scope.slotTimes = [];
+        tdprSlotsService.getSlotsTimes().then(function(response) {
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js:4]: Maybe resolve the slots times at the top level of the application so you can inject them everywhere?

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 3]
```diff
@@ -48,12 +65,28 @@ define(['angular', 'application/interviewer/tdprInterviewerModule'], function(an
                     }
                 }
             }
-            var startDate = $filter('date')(getDayOfTheWeek(new Date(), 0), "dd-MM-yyyy");
-            var endDate = $filter('date')(getDayOfTheWeek(new Date(), 4), "dd-MM-yyyy");
-            tdprSlotsService.updateSlots(slots, id, startDate, endDate);
+            var startDate = $filter('date')(getDayOfTheWeek(new Date(), relativeDayNumber), "dd-MM-yyyy");
+            var endDate = $filter('date')(getDayOfTheWeek(new Date(), 4 + relativeDayNumber), "dd-MM-yyyy");
+            tdprSlotsService.updateSlots(slots, id, startDate, endDate).then(function(){
+                            $state.go("tdpr.interviewer.home");
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js]: If we change the state here, do we need to set the submitSuccess flag below?

**Pull request merged**